src/Typedefs/Test/JSONTests.idr

```diff
@@ -132,6 +132,19 @@ nestedMu5Defs = [ dquotes "nestedMu5" |+| colon
             |++| taggedSumDef [ ("Nil", unitRef)
                               , ("Cons", prodDef [ref "nestedMu5", ref "NilCons"]) ]
 
+listOfDefsJSON : List Doc
+listOfDefsJSON = [ dquotes "bit" |+| colon |++| sumDef [ unitRef, unitRef ]
+                 , unitDef
+                 , dquotes "nibble" |+| colon |++| prodDef (replicate 4 $ ref "bit")
+                 , dquotes "byte" |+| colon |++| prodDef (replicate 2 $ ref "nibble")
+                 , dquotes "char" |+| colon |++| ref "byte"
+                 , dquotes "hash" |+| colon |++| ref "byte"
+                 , dquotes "transitionId" |+| colon |++| ref "byte"
+                 , dquotes "data" |+| colon |++| ref "byte"
+                 , dquotes "previous" |+| colon |++| ref "hash"
+                 , dquotes "rootTx" |+| colon |++| prodDef [ref "data", ref "previous"]
+                 ]
```

Comment:
I'd be in favour of aligning these on `|+|` and `|++|` to expose the tabular structure.